pkgs/development/tools/misc/gede/default.nix

```
@@ -1,31 +1,35 @@
 {
   mkDerivation,
   lib,
-  fetchurl,
+  fetchFromGitHub,
   makeWrapper,
   python3,
+  qtbase,
   qmake,
+  qtserialport,
   ctags,
   gdb,
 }:
 
 mkDerivation rec {
-  pname = "gede";
-  version = "2.18.3";
-
-  src = fetchurl {
-    url = "http://gede.dexar.se/uploads/source/${pname}-${version}.tar.xz";
-    sha256 = "sha256-RUl60iPa4XSlUilpYKaYQbRmLqthKHAvYonnhufjPsE=";
+  name = "gede";
+  version = "v2.22.1";
+
+  src = fetchFromGitHub {
+    owner = "jhn98032";
+    repo = name;
```

```suggestion
    repo = "gede";
```

use of name or pname is discouraged due to them being overridable

---

fixed
thanks for your comments